<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find asr_world_model)/log/rosconsole.config" />
    <node pkg="asr_world_model" type="asr_world_model" name="world_model" ns="env" output="screen">
        <rosparam file="$(find asr_world_model)/rsc/world_model_settings.yaml" />
        <rosparam file="$(find asr_world_model)/rsc/world_descriptions/world_description.yaml" />
        <!-- Needed to get bin_size and maxProjectionAngleDeviation -->
        <!-- Needed to get the dbfilename -->
        <rosparam file="$(find recognizer_prediction_ism)/launch/helpers/scene_recognition.yaml" />
        <!-- The file name for the itermediate_object weights, generated by the intermediate_object_generator (XXX will be replaced with dbfilename) -->
        <param name="intermediate_object_weight_file_name" value="$(find asr_world_model)/rsc/IntermediateObject_XXX.xml"/>
    </node>
</launch>